src/main/java/mate/academy/bookstore/service/ShoppingCartService.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.bookstore.service;
+
+import mate.academy.bookstore.dto.shopping.cart.ShoppingCartResponseDto;
+import mate.academy.bookstore.dto.shopping.cart.item.CartItemRequestDto;
+import mate.academy.bookstore.dto.shopping.cart.item.UpdateCartItemRequestDto;
+import mate.academy.bookstore.model.User;
+
+public interface ShoppingCartService {
+
+    void createShoppingCart(User user);
+
+    ShoppingCartResponseDto addToShoppingCart(CartItemRequestDto requestDto, Long userId);
+
+    ShoppingCartResponseDto getShoppingCart(Long userId);
+
+    ShoppingCartResponseDto updateShoppingCart(
+            Long userId, Long id, UpdateCartItemRequestDto requestUpdateDto);
```

Comment:
From the context of ShoppingCartService it's not obvious which `id` it is, so be more specific.
```suggestion
            Long userId, Long cartItemId, UpdateCartItemRequestDto requestUpdateDto);
```